src/lib/components/AccordionItem.svelte

```diff
@@ -28,6 +26,9 @@
 		}
 	}}
 >
+	{#if (showBody && $$slots.body)}
+		<svg xmlns="http://www.w3.org/2000/svg" height="17" width="17" class="chevron" transform={open ? "rotate(180)" : "rotate(0)"} viewBox="0 0 448 512"><path xmlns="http://www.w3.org/2000/svg" fill="#005eb8" d="M240.971 130.524l194.343 194.343c9.373 9.373 9.373 24.569 0 33.941l-22.667 22.667c-9.357 9.357-24.522 9.375-33.901.04L224 227.495 69.255 381.516c-9.379 9.335-24.544 9.317-33.901-.04l-22.667-22.667c-9.373-9.373-9.373-24.569 0-33.941L207.03 130.525c9.372-9.373 24.568-9.373 33.941-.001z"/></svg>
```

Comment:
Nice suggestion. There is one thing I found a bit non-ideal. If you pass a class in the Icon, which is in your component it does not have an effect on the Icon. This make sense, since the class is placed on the Icon and Svelte is looking for the class in the Icon component. But this approach works for example with Tailwind, because the classes there are global. This is why here I had to wrap the Icon in a div. This is no problem, but something to keep in mind. :)